Source/UI/NewUI/PaymentForm/ViewModel/DefaultPaymentViewModel.swift

```diff
@@ -27,7 +27,7 @@ class DefaultPaymentViewModel: PaymentViewModel {
        paymentFormStyle: PaymentFormStyle?,
        billingFormStyle: BillingFormStyle?,
        supportedSchemes: [Card.Scheme]) {
-    self.supportedSchemes = supportedSchemes
+    self.supportedSchemes = NSOrderedSet(array: supportedSchemes).map({ $0 as! Card.Scheme })
```

Comment:
compiles fine without this change

```suggestion
    self.supportedSchemes = supportedSchemes
```

Reply:
if you want to remove duplicates you can also remove the force cast
```suggestion
    self.supportedSchemes = Array(_immutableCocoaArray: NSOrderedSet(array: supportedSchemes))
```

Reply:
@harry-brown-cko i didn't find `_immutableCocoaArray` in swift. please explain more. thanks

Reply:
yeah was just testing stuff and xcode suggested that init. how about this instead? Saves the mapping step
```suggestion
    self.supportedSchemes = NSOrderedSet(array: supportedSchemes).array as! [Card.Scheme]
```

Reply:
Sorry to hop in, but whatever you go with could you please remove the forced cast? Removing duplicates is a caring fix if consumer mistakenly adds duplicate inputs, but if the price to that is force casting, we might want to review value of the feature.

I would add [OptionSet](https://www.hackingwithswift.com/example-code/language/how-to-create-a-custom-optionset) for consideration on this. There can be no duplicates if input is an option set

Reply:
I suggest 
```suggestion
    self.supportedSchemes = NSOrderedSet(array: supportedSchemes).array as? [Card.Scheme] ?? [.unknown]
```

Reply:
Looks better. Last question, why are we using `[.unknown]` as the fallback instead of just `[]`?

Reply:
As default is `.uknown`, should i change it?